v1.2/alter-sequence.md

```diff
@@ -4,13 +4,15 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v1.2:</span> The `ALTER SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `ALTER SEQUENCE` [statement](sql-statements.html) applies a schema change to a sequence.
+
+{{site.data.alerts.callout_info}}To understand how CockroachDB changes schema elements without requiring table locking or other user-visible downtime, see <a href="https://www.cockroachlabs.com/blog/how-online-schema-changes-are-possible-in-cockroachdb/">Online Schema Changes in CockroachDB</a>.{{site.data.alerts.end}}
 
 <div id="toc"></div>
 
 ## Required Privileges
```

Comment:
How do you check for privileges?

Reply:
Not quite sure… You can do `SELECT * FROM information_schema.table_privileges`, and see the schema there alongside with other tables.

I guess things should be tested here like "you can't call nextval() on a sequence if you don't have the UPDATE privilege on it"… Which probably fails now. That's something I should fix and put in the automated tests though… Sigh